genesyscloud/group/resource_genesyscloud_group_test.go

```diff
@@ -89,6 +89,39 @@ func TestAccResourceGroupBasic(t *testing.T) {
 
 				PreventPostDestroyRefresh: true,
 			},
+			{
+				// Update group
+				Config: generateUserWithCustomAttrs(testUserResourceLabel, testUserEmail, testUserName) + GenerateGroupResource(
+					groupResourceLabel1,
+					groupName,
+					strconv.Quote(groupDesc2),
+					strconv.Quote(typeOfficial), // Cannot change type
+					strconv.Quote(visMembers),
+					util.FalseValue,
+					"roles_enabled = true",
+					"owner_ids = []",
```

Comment:
I think the real test is if the owner_ids is not set at all, and a check on the API to confirm that there are no owner_ids.